### DISCUSSION TITLE
Should RFCs be moved into the documentation site?

### DISCUSSION THREAD

**Discussion by sarayourfriend** [WordPress/openverse#995] [Polls]
RFCs, which in our case includes essentially all project planning documents, have their own top-level `rfcs` directory. Should we move RFCs into the documentation site (https://docs.openverse.org) under a new "Project Planning" heading. Additionally, should we include the project planning process documentation (currently under `docs` directory) under that heading as well?

**Comment by zackkrida** [comment]
Excellent idea. I basically think all of our markdown files should be in the docs site at this point!

**Comment by zackkrida** [comment]
@WordPress/openverse-maintainers unless anyone objects to this change, I'm going to create an issue for its implementation.

**Reply by obulat** [reply]
I agree with moving RFCs (and all the md files) to the docs site. Thank you for creating the issue!

**Comment by zackkrida** [comment]
Issue here #1162.
